linux_os/guide/system/permissions/restrictions/sysctl_kernel_core_pattern_empty_string/bash/shared.sh

```diff
@@ -0,0 +1,60 @@
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_ubuntu,multi_platform_sle
+# reboot = true
+# strategy = disable
+# complexity = low
+# disruption = medium
+# Remediation is applicable only in certain platforms
+if [ ! -f /.dockerenv ] && [ ! -f /run/.containerenv ]; then
+
+# Comment out any occurrences of kernel.core_pattern from /etc/sysctl.d/*.conf files
+
+for f in /etc/sysctl.d/*.conf /run/sysctl.d/*.conf; do
```

Comment:
The problem is that the Ansible remediation is commenting out only offending occurrences of core_pattern, which means lines that have some value defined for the parameter. Here it would comment even a correct line. I'm not sure if it makes sense to invest much time on making them perfect. If they both bring the machine to a consistent compliant state, I believe it should be enough for now.

Reply:
OK, I agree. Let's leave it as it is now.